src/main/java/TesterVisitor.java

```diff
@@ -101,9 +105,76 @@ public Value visitVarReassignment(TesterParser.VarReassignmentContext ctx) {
 
     @Override
     public Value visitRequest(TesterParser.RequestContext ctx) {
-        return super.visitRequest(ctx);
+
+        String method = ctx.method().getText();
+        String endpoint = stripQuotes(ctx.endpoint().getText());
```

Comment:
```suggestion
        String endpoint = processString(ctx.endpoint().getText());
```
This way you also handle `${}` and `@{}`